docs/reference/certificates.md

```diff
@@ -1,85 +1,83 @@
-| etcd/ca.crt,key        | kubernetes-front-proxy-ca | For all ETCD-related functions   |
-| front-proxy-ca.crt,key | etcd-ca                   | for the [front-end proxy][proxy] |
-
-If you don't wish to copy these private keys to your API servers, you can generate all certificates yourself. 
-In that case, you will also need to make all of these:
+| ca.crt,key             | kubernetes-ca             | Kubernetes general CA            |
+| etcd/ca.crt,key        | etcd-ca                   | For all etcd-related functions   |
+| front-proxy-ca.crt,key | kubernetes-front-proxy-ca | For the [front-end proxy][proxy] |
 
+### All certificates
 
+If you don't wish to copy these private keys to your API servers, you can generate all certificates yourself. 
 
-[2]: For a liveness probe, if self-hosted
+Required certificates:
 
 | Default CN                    | Parent CA                 | O (in Subject) | kind                                   | hosts (SAN)                                 |
 |-------------------------------|---------------------------|----------------|----------------------------------------|---------------------------------------------|
-| kube-etcd                     | etcd-ca                   |                | server, client [<sup>1</sup>][etcdbug] | localhost, 127.0.0.1                        |
-| kube-etcd-peer                | etcd-ca                   |                | peer                                   | <hostname>, <Host IP>, localhost, 127.0.0.1 |
+| kube-etcd                     | etcd-ca                   |                | server, client [<sup>1</sup>][etcdbug] | `localhost`, `127.0.0.1`                        |
+| kube-etcd-peer                | etcd-ca                   |                | peer                                   | `<hostname>`, `<Host_IP>`, `localhost`, `127.0.0.1` |
 | kube-etcd-healthcheck-client  | etcd-ca                   |                | client                                 |                                             |
 | kube-apiserver-etcd-client    | etcd-ca                   | system:masters | client                                 |                                             |
-| kube-apiserver                | kubernetes-ca             |                | server                                 | <hostname>, <Host IP>, <advertise IP> , [1] |
+| kube-apiserver                | kubernetes-ca             |                | server                                 | `<hostname>`, `<Host_IP>`, `<advertise_IP>`, `[1]` |
 | kube-apiserver-kubelet-client | kubernetes-ca             | system:masters | client                                 |                                             |
 | front-proxy-client            | kubernetes-front-proxy-ca |                | client                                 |                                             |
 
+[1]: `kubernetes`, `kubernetes.default`, `kubernetes.default.svc`, `kubernetes.default.svc.cluster`, `kubernetes.default.svc.cluster.local`
```

Comment:
Changed markup for this and other notes because they weren't rendering at all in the original.